frontend/src/models/auth.ts

```diff
@@ -30,14 +38,11 @@ export default createModel({
 
       if (!user) {
         const storedUser = window.localStorage.getItem(USER_KEY)
-        const backendCredentials = parse(window.location.href, true).query
-
-        console.log('INIT AUTH', { backendCredentials, storedUser })
-
-        if (backendCredentials.username && backendCredentials.authHash) user = backendCredentials
+        console.log('INIT AUTH', { storedUser })
         if (storedUser) user = JSON.parse(storedUser)
       }
-      if (user) {
+      if (user?.username) {
+        console.log('WE HAVE A USER', user)
```

Comment:
You want this kind of log in production?

Reply:
Naw.... I'll take that out - thanks.